account_invoice_import/wizard/account_invoice_import_error.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+# Copyright (C) 2021 Essent <http://www.essent.be>
+# @author Stefan Rijnhart <stefan.rijnhart@dynapps.be>
+# @author Pieter Paulussen <pieter.paulussen@dynapps.be>
+# @author Robin Conjour <r.conjour@essent.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning as UserError
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class AccountInvoiceImportError(models.TransientModel):
+    _name = 'account.invoice.import.error'
+    _description = 'Wizard to show errors after completion'
+
+    @api.multi
+    def view_invoice(self):
+        """ Method to redirect to the invoice view. """
+        self.ensure_one()
+        invoice_id = self.env.context.get('invoice_id')
+        if not invoice_id:
```

Comment:
`if 'invoice_id'  not in self.env.context or not self.env.context['invoice_id']:`

Reply:
Wait, isn't this what `dict.get` is for?